src/js/core/methods/GetAccountInfo.js

```diff
@@ -62,6 +62,7 @@ export default class GetAccountInfo extends AbstractMethod {
                 { name: 'contractFilter', type: 'string' },
                 { name: 'gap', type: 'number' },
                 { name: 'marker', type: 'object' },
+                { name: 'defaultAccountType', type: 'boolean' },
```

Comment:
shouldn't this be `string` instead of `boolean`?

Reply:
good cath, fixed